docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md

```diff
@@ -245,3 +245,10 @@ with `cert-manager.io/v1`
   ```yaml
   serviceAccountName: manager
   ```
+
+## Percentage String or Int API input will fail with a string different from an integer with % appended.
```

Comment:
As this document was afaik previously mostly used to document steps for infra providers to migrate from v1alpha3 to v1alpha4. Is there something infra providers have to change for this? Or just to make sure the infra providers are aware of the change?

Reply:
The latter. I can drop it if folks think is not appropriate to have it in this doc.

Reply:
Both options are fine for me. Just wanted to be absolutely sure that I don't miss something for CAPO.